.travis.yml

```diff
@@ -28,6 +28,8 @@ install:
  - curl -sSf https://sh.rustup.rs | sh -s -- -y
  - export PATH=$HOME/.cargo/bin:$PATH
  - rustc --version
+ # ./tools/fetch_cargo_crates.py depends on this subcommand (https://github.com/ehuss/cargo-clone-crate)
+ - cargo install cargo-clone-crate
```

Comment:
This should be part of build_third_party.py. It's the analog of yarn for rust - and yarn is called there.